common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5138,6 +5138,15 @@ public static enum ConfVars {
     LLAP_WEBUI_SPNEGO_PRINCIPAL("hive.llap.webui.spnego.principal", "",
         "The LLAP WebUI SPNEGO service principal. Configured similarly to\n" +
         "hive.server2.webui.spnego.principal"),
+    HIVE_SERVER2_WEBUI_HTTP_COOKIE_MAX_AGE("hive.server2.ui.http.cookie.max.age", "86400s",
```

Comment:
this should go right after the other hs2 web ui options:
https://github.com/apache/hive/blob/b13f529d944dc4c72d02d18e63757622985ada28/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java#L4018

Reply:
Done

Reply:
ack